test/dynamo/test_subgraphs.py

```diff
@@ -580,7 +580,7 @@ def fn(x, it):
 
     def test_enumerate_not_break_graph(self):
```

Comment:
This enumerate is important! It's testing for enumerate graph breaks. Be wary of JIT and Dynamo tests and check the name of the test

Reply:
Thank you, reverted

Reply:
There is a ruff rule here you are likely planning on enabling. Can you add the corresponding noqa to this line to prevent this issue later on?